### PR TITLE
remove usage of deprecated bootstrap flags

### PR DESCRIPTION
**Pull request by BenTheElder**
This will help simplify the current bootstrap and the port to go (#3370).

Fixes #4456

### CODE REVIEW

File: jenkins/builder.sh [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-
-set -o errexit
-set -o nounset
-set -o pipefail
-set -o xtrace
-
-# Find test scripts.
-script="$(dirname "${0}")/bootstrap.py"
-
-"${script}" --job="${1%.sh}" --repo=kubernetes/kubernetes --pull="${PULL_REFS:-}"
```
Comment (BenTheElder) [inline jenkins/builder.sh]: I couldn't find anything using this @fejta @spxtr (?)

**Comment by krzyzacy** [comment]
push a new kubeadm image?

**Comment by BenTheElder** [comment]
@krzyzacy perhaps I should wait on https://github.com/kubernetes/test-infra/pull/4449 ?

**Comment by BenTheElder** [comment]
Added do-not-merge until we sort out updating the kubeadm images, PTAL at the other changes though.

**Comment by BenTheElder** [comment]
Discussed offline with @krzyzacy - we will return to this tomorrow morning when we have more time to monitor for breakage.

**Comment by BenTheElder** [comment]
@krzyzacy bumped, PTAL

File: jenkins/builder.sh (hunk 1 above)
Comment (BenTheElder) [reply jenkins/builder.sh]: fejta responded on slack, doesn't remember this file at all and thinks it's probably just a "zombie" so deleting it should be fine.

File: jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-security-pull.yaml [hunk 2]
```diff
@@ -55,8 +55,7 @@
         git clone https://github.com/kubernetes/test-infra -b master
         './test-infra/jenkins/bootstrap.py' \
             --job='{job-name}' \
-            --pull="${{PULL_REFS}}" \
-            --repo='{repo-name}' \
+            "--repo=k8s.io/{repo-name}=${{PULL_REFS}}" \
```
Comment (krzyzacy) [inline jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-security-pull.yaml]: (I feel like the security repo is not under k8s.io, and I also feel like the presubmit for security repo is broken (which might be good?) )

**Comment by krzyzacy** [comment]
cc @luxas ^^ some refactoring and cleans up deprecated bootstrap flag that kubeadm image is still using

LGTM except for the security repo prefix

File: jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-security-pull.yaml (hunk 2 above)
Comment (BenTheElder) [reply jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-security-pull.yaml]: whoops! fixed. probably pasted from the kubernetes pull config :roll_eyes:

**Pull request merged**